source/_integrations/http.markdown

```diff
@@ -168,6 +174,10 @@ If you want to use Home Assistant to host or serve static files then create a di
 
 </div>
 
+### Symbolic Links
+
+If you use symbolic links in your `www/` folder, then you will need to set `follow_symlinks: True` in the configuration file.
```

Comment:
Let's add this text to the option directly instead of creating a single line chapter.